test/evergreen.yml

```diff
@@ -2854,9 +2860,23 @@ buildvariants:
   run_on:
   - ubuntu2004-test
   expansions:
-    test_env_vars: LD_PRELOAD=/usr/lib/x86_64-linux-gnu/libeatmydata.so PATH=/opt/mongodbtoolchain/v3/bin:$PATH LD_LIBRARY_PATH=$(pwd)/.libs top_srcdir=$(pwd)/.. top_builddir=$(pwd)
+    test_env_vars:
+      LD_PRELOAD=/usr/lib/x86_64-linux-gnu/libeatmydata.so
+      PATH=/opt/mongodbtoolchain/v3/bin:$PATH
+      LD_LIBRARY_PATH=$(pwd)/.libs:$(git rev-parse --show-toplevel)/TCMALLOC_LIB/lib
```

Comment:
Is it possible to move this definition below `top_srcdir` & `top_builddir`, so you can define `LD_LIBRARY_PATH` as :
```
LD_LIBRARY_PATH=$(pwd)/.libs:$(top_srcdir)/TCMALLOC_LIB/lib
```
It would be ideal to reduce the number of definitions/interpretations of the top source directory, keeping everything consistent (particularly if we ever need to change the definition of `top_srcdir`). 
This applies to all below usages also.

Reply:
Done! Let me know if you it looks good for you.